unitOneAssessmentPractice.js

```diff
@@ -3,39 +3,100 @@ let assert = require('assert')
 // Question One:
 
 // Write a function called double that doubles a number
-
-// Uncomment out the next line to test your solution
+function double(num) {
+  return num*2;
+}
+double(14);
+// // Uncomment out the next line to test your solution
 // runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
+function containsSeven(num) {
+ let newNum = num.toString();
+ for (let i = 0; i < newNum.length; i++) {
+   if (newNum[i] === "7") {
+    return true;
+   } else {
+    i++;
```

Comment:
Instead of i++, you could put return false here. That way the function will return false once 7 is found in the string.

Reply:
I couldn't seem to get it to work with just a return false, but I have pushed updated code to account for different types of number inputs. i think this may work better than what i had originally